test/resize.contentWidth.html

```diff
@@ -40,7 +40,8 @@
                 sendMessage('width')
                 break
               case '2':
-                ok(3000 < parseInt(messageData.width, 10), 'iFrame width<3000')
+                ok(3000 < messageData.width, 'iFrame width<3000')
+                ok(messageData.width.toString() !== messageData.width, 'width is number')
```

Comment:
This is the only spot where I added an additional check, because the original check passed whether or not `message.width` was a string, due to implicit type coercion.

I considered utilizing coercion to do this detection; eg
```ts
ok(3000 < message.width + 1000, 'iFrame width < 3000)
// old, bad output '30011000'
// new, good output 4001
```

However, this seemed a little too magical, so I opted to add an explicit type-ish check.